manifests/config.pp

```diff
@@ -37,13 +37,15 @@
   $cache_dirs                    = $::squid::cache_dirs,
   $cache                         = $::squid::cache,
   $extra_config_sections         = $::squid::extra_config_sections,
+  $squid_bin_path                      = $::squid::squid_bin_path,
```

Comment:
@jocelynthode If you could fix up this indentation, I think we're otherwise good to merge.

Reply:
Done